.evergreen/run-tests.sh

```diff
@@ -13,27 +13,21 @@ AUTH=${AUTH:-noauth}
 SSL=${SSL:-nossl}
 MONGODB_URI=${MONGODB_URI:-}
 TESTS=${TESTS:-}
-PHP_IS_MATRIX_TESTING=${PHP_IS_MATRIX_TESTING:-false}
+IS_MATRIX_TESTING=${IS_MATRIX_TESTING:-false}
 
 # For matrix testing, we have to determine the correct driver version
-if [ "$PHP_IS_MATRIX_TESTING" == "true" ]; then
-   set_extension_version () {
-      case $1 in
-         '4.4')
-             export DRIVER_VERSION='1.8.2'
-             ;;
-         '4.2')
-            export DRIVER_VERSION='1.6.1'
-            ;;
-         '4.0')
-            export DRIVER_VERSION='1.5.5'
-            ;;
-      esac
-   }
-
-   set_extension_version "${DRIVER_MONGODB_VERSION}"
-
-   echo $DRIVER_VERSION
+if [ "$IS_MATRIX_TESTING" == "true" ]; then
+   case "${DRIVER_MONGODB_VERSION}" in
+      '4.4')
+          export DRIVER_VERSION='1.8.2'
```

Comment:
Won't this clobber the definitions in https://github.com/mongodb/drivers-matrix-testing/pull/6?

Reply:
Ah, good point. We probably shouldn't use `export`, but this should be fine without. Alternatively, I can change this from `DRIVER_VERSION` to `EXTENSION_VERSION` (which would be more appropriate anyways). I'll just have to remember to update the config file as well once this gets merged up into a branch with evergreen config present.

Reply:
I assume this is just about relaying information to `install-dependencies.sh`, which is called a few lines later. In that case, `EXTENSION_VERSION` seems fine. Using distinct environment variables feels cleaner and could avoid a WTF down the line.

As for merging up to another branch, perhaps you want to lump this todo item in with the simple-phpunit item in https://github.com/mongodb/mongo-php-library/pull/810/files#r574301076?

Reply:
Yep, this will produce conflicts anyways.